### PR TITLE
Add Spanish translation for redis/sessions.md

### PR DESCRIPTION
**Pull request by alemohamad**
Add Spanish translation for redis/sessions.md

### CODE REVIEW

File: docs/redis/sessions.es.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+La implementación predeterminada de `RedisSessionsDelegate` almacenará [`SessionData`](https://api.vapor.codes/vapor/documentation/vapor/sessiondata) como un simple valor de cadena JSON usando `Codable`.
+
+A menos que implementes el método [`makeRedisKey(for:)`](https://api.vapor.codes/redis/documentation/redis/redissessionsdelegate/makerediskey(for:)-5nfge) en tu propio `RedisSessionsDelegate`, `SessionData` se almacenará en Redis con una clave que antepone el `SessionID` con `vrs-` (**V**apor **R**edis **S**essions).
+
+Por ejemplo: `vrs-Hbxozx8rTj+XXGWAzOhh1npZFXaGLpTWpWCaXuo44xQ=`
+
+## Registrando un Delegado Personalizado
+
+Para personalizar cómo se lee y se escriben los datos en Redis, registra tu propio objeto `RedisSessionsDelegate` de la siguiente manera:
```
Comment (jacostaf10) [inline docs/redis/sessions.es.md:28]:
```suggestion
Para personalizar cómo se leen y se escriben los datos en Redis, registra tu propio objeto `RedisSessionsDelegate` de la siguiente manera:
```

File: docs/redis/sessions.es.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+app.sessions.use(.redis(delegate: CustomRedisSessionsDelegate()))
+```
+
+## RedisSessionsDelegate
+
+> Documentación de la API: [`RedisSessionsDelegate`](https://api.vapor.codes/redis/documentation/redis/redissessionsdelegate)
+
+Un objeto que cumple con este protocolo puede usarse para cambiar cómo `SessionData` se almacena en Redis.
+
+Solo dos métodos son requeridos para ser implementados por un tipo que cumpla con el protocolo: [`redis(_:store:with:)`](https://api.vapor.codes/redis/documentation/redis/redissessionsdelegate/redis(_:store:with:)) y [`redis(_:fetchDataFor:)`](https://api.vapor.codes/redis/documentation/redis/redissessionsdelegate/redis(_:fetchdatafor:)).
```
Comment (jacostaf10) [inline docs/redis/sessions.es.md:46]:
```suggestion
Solo dos métodos requieren de implementación en un tipo que cumpla con el protocolo: [`redis(_:store:with:)`](https://api.vapor.codes/redis/documentation/redis/redissessionsdelegate/redis(_:store:with:)) y [`redis(_:fetchDataFor:)`](https://api.vapor.codes/redis/documentation/redis/redissessionsdelegate/redis(_:fetchdatafor:)).
```

**Review by jacostaf10 — CHANGES_REQUESTED** [review]
Just to improve readability, the rest is okay.

**Comment by alemohamad** [comment]
Updated changes!

**Review by jacostaf10 — APPROVED** [review]
Ready!

**Pull request merged**